Editors/ViewModels/SchematicsViewModel.cs

```diff
@@ -147,24 +148,29 @@ private void SimulationThread(object state)
         }
 
         public bool IsSimulationRunning => simulation != null;
-        public IEnumerable<Entities.Entity> Entities
+        public List<Entities.Entity> Entities
```

Comment:
why did u decide to use a list

Reply:
idr, can probably be reverted. the performance chance would be minimal anyways because this new method has this whole part take like 0.6 seconds to do